src/main/groovy/com/zlad/gradle/avrohugger/GeneratorFactory.groovy

```diff
@@ -17,6 +17,7 @@ class GeneratorFactory implements Serializable {
     Generator create() {
         final format = sourceFormat.toAvrohuggerSourceFormat()
         final namespaces = ScalaConversions.convert(customNamespaces)
+        final targetScalaPartialVersion = null
```

Comment:
First, thanks for the nice update.

Then instead of using avrohugger's scala version (default), we should use scala version from the project dependencies. There is already property provider `ScalaVersionBelow2_11` which could be updated to get the  version and pass here.

Neverthless, we can go with this for now and I will try to do the follow up update as soon as possible.

Reply:
Fixed in 01c18b67fac59a2316f736053b360bedc04dd5c4, released as `0.7.0`.